src/user.ts

```diff
@@ -146,6 +146,36 @@ export class UserApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /**
```

Comment:
Your other function blocks have `/*` and this one is `/**`... I don't care which, but pick one - Fair?